influx/users.go

```diff
@@ -21,7 +21,7 @@ func (c *Client) Add(ctx context.Context, u *chronograf.User) (*chronograf.User,
 			return nil, err
 		}
 	}
-	return c.Get(ctx, u.Name)
+	return c.Get(ctx, chronograf.UserQuery{Name: &u.Name})
```

Comment:
if influx Users share the same Bolt store, the same comment from `enterprise` applies here about `influx` Users and these methods – they'll need to be augmented to reflect the enhanced User model since `Name` is sufficient to guarantee uniqueness

Reply:
Ditto the statement above.